doc/tutorial/01_introduction/index.md

```diff
@@ -48,120 +49,91 @@ Then print the vector with seqan3::debug_stream. Does your program also work wit
 \endassignment
 \solution
-\snippet introduction_read_fasta.cpp read
+\snippet introduction_dna4.cpp debug
 \endsolution
 
-Note that the same code can also read FASTQ files and the `qual` variable will not be empty then. If you like, try it!
+# Some general notes that might help to dive in to SeqAn
 
-\note
-SeqAn3 uses `snake_case` for almost everything, also class names. Only C++ concepts are named using `CamelCase`.
+## SeqAn and the stl
 
-# Align two sequences
+In contrast to the former version of SeqAn (2.x.x releases), we try to be very close to the standard and all other
+sata structures and algorithms should work on stl data structures as well.
 
-We have two sequences from the file above now – so let us align them.
-The pairwise sequence alignment is one of the core algorithms in SeqAn and used by several library components
-and apps. It is strongly optimised for speed and parallel execution while providing exact results and a
-generic interface.
+Analog to the stl, SeqAn3 uses `snake_case` everywhere.
 
-\snippet introduction_align.cpp alignment
-
-The algorithm returns a range of result objects – which is the reason for the loop here (in this case the range
-has length 1). Instead of passing a single pair of sequences, we could give a vector of sequence pairs to the
-algorithm which then executes all alignments in parallel and stores the results in various seqan3::alignment_result
-objects. The second argument to seqan3::align_pairwise is the *configuration* which allows you to specify
-a lot of parameters for the alignment computation, for instance score functions, banded alignment and whether
-you wish to compute a traceback or not. The configurations have their own namespace seqan3::align_cfg and can
-be combined via the logical OR operator (`|`) for building combinations. Check out the alignment tutorial if you want
-to learn more.
-
-\note
-We encourage you to avoid declaring `using namespace seqan3;`. This has the additional benefit of easily distinguishing
-between library features and standard C++. The only exception are string literals, where we often use
-`using namespace seqan3::literals` for convenience.
+## Modern C++
 
 \note
 We use a lot of Modern C++ in SeqAn so some things might look alien at first,
 e.g. type templates are used like ordinary types in many situations (no `<>`).
 We also always use `{}` to initialise objects and not `()` which is only used for function calls.
 In general, the style should be much easier for newcomers.
 
+## Avoid `using namespace seqan3;`
+
+We encourage you to avoid declaring `using namespace seqan3;`. This has the additional benefit of easily distinguishing
+between library features and standard C++. The only exception are string literals, where we often use
+`using namespace seqan3::literals` for convenience.
+
+All our tests also avoid `using namespace seqan3;` so we can't ensure the safety of doing so.
```

Comment:
Not sure if helpful. Could also be read as "we are too lazy to test it, so don't use it" instead of "it is bad style, so don't do it".

Reply:
haha yes.. I'll remove it again. Or would you just rephrase?